Week3/src/Week3/taming_bronze.java

```diff
@@ -0,0 +1,78 @@
+//Leeland Zhang
+//Took me like 90 minutes
+//I got 5
+//This problem is like a 9 i guess.
+//I really just gave up.
+//package Week3;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+public class taming_bronze 
+{
+	public static void main(String args[]) throws IOException
+	{
+		Scanner sc = new Scanner(new File("taming.in"));
+		PrintWriter pr=new PrintWriter(new FileWriter("taming.out"));
+		//Scanner sc = new Scanner(System.in);
+		int n=sc.nextInt();
+		int[] arr=new int[n];
+		for(int i=0; i<n; i++)
+		{
+			arr[i]=sc.nextInt();
+		}
+//-----------------------------------------------------------------------
+		boolean neg=false;
+		int count=0;
+		int temp=0;
+		int min=1; 
+		int max=1;
+		for(int i=1; i<n; i++)
+		{
+				if(arr[i]==-1)
+					count++;

```

Comment:
You indentation isn't consistent. Try use 4 space indentation. Using "tab" (8 spaces) for indentation is fine. Either way, you need make sure you are consistent. To be effective, during practice, you need develop the habit of always fix your indentation.